### PR TITLE
107 make rooms searchable by room typ and tags

### PR DESCRIPTION
**Pull request by MartinPreiss**

### CODE REVIEW

**Review by cedric-lorenz — DISMISSED** [review]

**Review by cedric-lorenz — APPROVED** [review]

**Review by cedric-lorenz — DISMISSED** [review]

File: app/models/room.rb [hunk 1]
```diff
@@ -35,7 +35,11 @@ def to_geojson
   end
 
   def self.searchable_attributes
-    %w[number full_name]
+    %w[floor number full_name room_types.name tags.name]
```
Comment (elenagensch) [inline app/models/room.rb:38]:
```suggestion
    %w[number full_name room_types.name tags.name]
```

File: app/models/room.rb [hunk 2]
```diff
@@ -35,7 +35,11 @@ def to_geojson
...
+  end
+
+  def self.searchable_relations
+    [:tags, :room_types]
```
Comment (elenagensch) [inline app/models/room.rb:42]:
```suggestion
    [:floor, :tags, :room_types]
```

**Review by elenagensch — CHANGES_REQUESTED** [review]
floor is not a column (after dev-merge), please adjust searchable_attributes and relations

File: app/models/room.rb [hunk 3]
```diff
@@ -35,7 +35,11 @@ def to_geojson
   end
 
   def self.searchable_attributes
-    %w[number full_name]
+    %w[number full_name room_types.name tags.name floors.name]
+  end
+
+  def self.searchable_relations
+    [:floors, :tags, :room_types]
```
Comment (elenagensch) [inline app/models/room.rb:42]:
```suggestion
    [:floor, :tags, :room_types]
```

**Review by elenagensch — APPROVED** [review]

**Pull request merged**